omod/src/main/webapp/fragments/systemadmin/accounts/userFormFields.gsp

```diff
@@ -128,6 +128,16 @@
        ng-model="uuidUserMap['${userUuid}'].userProperties.forcePassword" />${ ui.message("adminui.account.user.forcePasswordChange") }
 </p>
 
+<% customUserFragments.each { sect -> %>
```

Comment:
Since the other one one has "View", shouldn't this have "Edit"?

Reply:
replaced with "Edit"